src/FunctionConfig.ts

```diff
@@ -94,4 +98,20 @@ export class FunctionConfig {
             throw new Error(localize('functionJsonParseError', 'Failed to parse function.json: {0}', errMessage));
         }
     }
+
+    public get inBinding(): IFunctionBinding {
+        if (!this._inBinding) {
+            throw this._noInBindingError;
+        } else {
+            return this._inBinding;
+        }
+    }
```

Comment:
Is this primarily for the tests below? I see no mention of `this.inBinding` in this file. 
I'm mainly confused between the `const inBinding`  in the diff, and the above getter function.

Reply:
inBinding is a public member and is used outside this file. Before this change, we required it to be defined. Now, we don't require it to be defined _unless you try to access it_. This is because sometimes the code uses other parts of TemplateData and doesn't really need inBinding